gitlab4j-api/src/main/java/org/gitlab4j/api/JobApi.java

```diff
@@ -291,6 +291,19 @@ public Stream<Job> getJobsStream(Object projectIdOrPath, long pipelineId, Boolea
         return (getJobsForPipeline(projectIdOrPath, pipelineId, getDefaultPerPage(), includeRetried).stream());
     }
 
+    /**
+     * Retrieve the job corresponding to the <code>$CI_JOB_TOKEN</code> environment variable (Using a {@link org.gitlab4j.models.Constants.TokenType#JOB_TOKEN} authentication).
+     *
+     * <pre><code>GitLab Endpoint: GET /job</code></pre>
+     *
+     * @return a single job
+     * @throws GitLabApiException if any exception occurs during execution
+     */
+    public Job getJob() throws GitLabApiException {
+        Response response = get(Response.Status.OK, null, "job");
```

Comment:
I would recommend checking that a JOB_TOKEN was set in gitlabApi and throwing a meaningful exception if not.

Reply:
Added with https://github.com/gitlab4j/gitlab4j-api/pull/1236